package.json

```diff
@@ -8,15 +8,19 @@
   "scripts": {
     "build": "./scripts/package.sh --local",
     "commit": "git-cz",
-    "checkcommit": "node ./scripts/commit-lint.js",
+    "checkcommit": "echo 'Please use check-commit as this script is deprecated'",
+    "check-commit": "node ./scripts/commit-lint.js",
+    "checkformat": "echo 'Please use check-format as this script is deprecated'",
+    "check-format": "scripts/check-format.sh",
+    "checkimports": "echo 'Please use check-imports as this script is deprecated'",
+    "check-imports": "node ./scripts/check-imports.js",
+    "checkversions": "echo 'Please use check-versions as this script is deprecated'",
+    "check-versions": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/check-versions.ts",
```

Comment:
if the said update is approved by the Nx team, please let me know if the older scripts (the ones without hyphens should be retained)